english/eat-tacos/eat-delicious-tacos

```diff
@@ -0,0 +1,7 @@
+You found a place in the street, you were still hungry and you didn't have a lot of money. Then you decided to eat in that place.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.